docs/Policies/App Policies/index.md

```diff
@@ -0,0 +1,244 @@
+
+## App Service Fields
+
+These fields are used if the application is either be a provider or consumer of app service data. See our [App Service Provider Guide](https://smartdevicelink.com/en/guides/core/app-service-provider-guide/) for more information on developing an application with these capabilities.
+
+### Example Entry
+
+```json
+"app_policies": {
+    "123abc": {
+        "keep_context": false,
+        "steal_focus": false,
+        "priority": "NONE",
+        "default_hmi": "NONE",
+        "groups": ["Base-4", "AppServiceProvider", "AppServiceConsumer"],
+        "RequestType": [],
+        "RequestSubType": [],
+        "app_services": {
+            "MEDIA": {
+                "service_names": ["SDL Music", "App Music"],
+                "handled_rpcs": [{"function_id": 41}]
+            },
+            "NAVIGATION": {
+                "service_names": ["SDL Navigation", "App Navigation"],
+                "handled_rpcs": [{"function_id": 39}]
+            },
+            "WEATHER": {
+                "service_names": ["SDL Weather", "App Weather"],
+                "handled_rpcs": []
+            }
+        },
+        "allow_unknown_rpc_passthrough": true
+    },
+    ...
+}
+```
+
+### app_services
+
+This field defines the properties of the app services that this application is allowed to publish. Each key within this object corresponds to an app service type (ex. "MEDIA").
+
+* `service_names` - Defines a list of potential service names that an app can publish with this service type. 
+    * If the `serviceName` field in the app's `PublishAppService.appServiceManifest` does not match one of the values in this list, the message will be rejected.
+    * If omitted, no restrictions will be applied to the `serviceName` parameter.
+* `handled_rpcs` - This field defines a list of permissions for RPCs that an app service could intercept via the [RPC passing](https://smartdevicelink.com/en/guides/core/app-service-provider-guide/#rpc-passing) feature. 
+    * If the `handledRPCs` field in the app's `PublishAppService.appServiceManifest` includes a function ID that is not in this list, the message will be rejected. 
+    * This field is mandatory, but can be empty.
+    * Subfields:
+        * `function_id` - The [function ID](https://github.com/smartdevicelink/sdl_core/blob/master/src/components/interfaces/MOBILE_API.xml) of the potential handled RPC. Can be a function ID which was introduced after Core's local RPC spec version.
+
+### allow_unknown_rpc_passthrough
+
+This flag can be used to allow an application to pass RPCs which are unknown to Core (functions which were defined after Core's local API version) to an app service provider. Otherwise, such messages from this application will be rejected. Defaults to `false`.
```

Comment:
>Defaults to `false`.

Can you please be more specific here?  For example, "This flag defaults to `false`."